vivarium/core/process.py

```diff
@@ -544,9 +604,10 @@ def update(
                 should be computed.
             states: The pre-update state of the simulation.
         """
-        self.parent.send((interval, states))
+        args = (interval, states)
+        self.send_command('_next_update', args)
 
-    def get(self) -> Update:
```

Comment:
no more `.get()`? Does this break an prior work?

Reply:
This is in `ParallelProcess`, so I think this is probably only used inside Vivarium Core. I doubt any clients are using this, but I'm not sure. That said, I haven't checked to make sure I've updated all the Vivarium Core usages yet.

Also note that this is a draft PR because I think we should discuss on #197 whether we even want to take this approach.